iocell.8.txt

```diff
@@ -1196,6 +1196,17 @@ PROPERTIES
     Default: off
     Source: cpuset(1)
 
+  volumes=/host/dir[:/jail/dir[:mount_options]]
+
+  Mounts and unmounts directories from the host into the jail root using
+  nullfs(5).
+  'mount_options' are passed direcly to the mount command and defaults to "rw"
```

Comment:
i do not agree. i can tell you from experience using this a lot in production that you want 'rw' 99% of the time. in fact the only cases i wanted 'ro' so far was for single file volumes that are config and should not be changable from within the jail. lots of services will fail to even start if they can't write to a directory they need access to and won't tell you necessarily that it was a permission problem.